UnityProjects/MRTKDevTemplate/Assets/Scripts/GridSqueezer.cs

```diff
@@ -50,11 +50,11 @@ public GridLayoutGroup GridLayoutGroup
         }
 
         [SerializeField]
-        [Tooltip("The RectTransform to fit the collider onto.")]
+        [Tooltip("The AttachedRectTransform to fit the collider onto.")]
```

Comment:
What's the reason for this change?

Reply:
Also curious is this script used in any of the samples? I can't find anywhere it is used

Reply:
OOps...I had doe a find-and-replace to address some other "RectTransform" comments.  See RectTransformColliderFitter.cs...in this file there's a property called "AttachedRectTransform", but had comments referring to a property called "RectTransform"